Makefile

```diff
@@ -0,0 +1,646 @@
+# Update VirulenceFinder database
+# -----------------------------
+VIRULENCEFINDERDB_DIR := $(ASSETS_DIR)/virulencefinder_db
+KMA_DIR := $(ASSETS_DIR)/kma
+
+update_virulencefinder_db: $(VIRULENCEFINDERDB_DIR)/stx.name
+
+$(ASSETS_DIR)/virulencefinder_db/stx.name: | check-and-reinit-git-submodules
+	$(call log_message,"Starting update of VirulenceFinder database")
+	cd $(ASSETS_DIR)/kma \
+	&& make \
+	&& cd $(VIRULENCEFINDERDB_DIR) \
+	&& export PATH=$(ASSETS_DIR)/kma:$$PATH \
+	&& singularity exec	--bind $(MNT_ROOT) $(CONTAINER_DIR)/pythonScripts.sif \
+		python3 INSTALL.py \
+		$(KMA_DIR)/kma_index |& tee -a $(INSTALL_LOG) \
+	&& cd $(ASSETS_DIR)/resfinder_db \
+	&& singularity exec --bind $(MNT_ROOT) $(CONTAINER_DIR)/pythonScripts.sif \
+		python3 INSTALL.py \
+		$(KMA_DIR)/kma_index |& tee -a $(INSTALL_LOG) \
+	&& cd $(ASSETS_DIR)/pointfinder_db \
+	&& singularity exec --bind $(MNT_ROOT) $(CONTAINER_DIR)/pythonScripts.sif \
+		python3 INSTALL.py \
+		$(KMA_DIR)/kma_index |& tee -a $(INSTALL_LOG)
+
+# ==============================================================================
+# Download, index and prep reference genomes for organisms
+# ==============================================================================
+
+# -----------------------------
+# S. aureus
+# -----------------------------
+saureus_all: saureus_download_reference \
+	saureus_index_reference \
+	saureus_download_prodigal_training_file \
+	saureus_download_cgmlst_schema \
+	saureus_unpack_cgmlst_schema \
+	saureus_prep_cgmlst_schema
+
+SAUR_GENOMES_DIR := $(ASSETS_DIR)/genomes/staphylococcus_aureus
+SAUR_CGMLST_DIR := $(ASSETS_DIR)/cgmlst/staphylococcus_aureus
+SAUR_REFSEQ_ACC := NC_002951.2
+
+
+saureus_download_reference: $(SAUR_GENOMES_DIR)/$(SAUR_REFSEQ_ACC).fasta
+
+$(SAUR_GENOMES_DIR)/$(SAUR_REFSEQ_ACC).fasta:
+	$(call log_message,"Downloading S. aureus reference genome ...")
+	mkdir -p $(SAUR_GENOMES_DIR) \
+	&& cd $(SCRIPT_DIR) \
+	&& singularity exec --bind $(MNT_ROOT) $(CONTAINER_DIR)/pythonScripts.sif \
+		python3 bin/download_ncbi.py \
+		-i $(SAUR_REFSEQ_ACC) \
+		-o $(SAUR_GENOMES_DIR) |& tee -a $(INSTALL_LOG) \
+
+
+saureus_index_reference: $(SAUR_GENOMES_DIR)/$(SAUR_REFSEQ_ACC).fasta.bwt
+
+$(SAUR_GENOMES_DIR)/$(SAUR_REFSEQ_ACC).fasta.bwt: $(SAUR_GENOMES_DIR)/$(SAUR_REFSEQ_ACC).fasta
+	$(call log_message,"Indexing S. aureus reference genome ...")
+	cd $(SAUR_GENOMES_DIR) \
+	&& singularity exec --bind $(MNT_ROOT) $(CONTAINER_DIR)/bwakit.sif \
+		bwa index $$(basename $<) |& tee -a $(INSTALL_LOG)
+
+
+saureus_download_prodigal_training_file: $(PRODIGAL_TRAINING_DIR)/Staphylococcus_aureus.trn
+
+$(PRODIGAL_TRAINING_DIR)/Staphylococcus_aureus.trn:
+	$(call log_message,"Downloading S. aureus prodigal training file ...")
+	mkdir -p $(PRODIGAL_TRAINING_DIR) \
+	&& cd $(PRODIGAL_TRAINING_DIR) \
+	&& wget https://raw.githubusercontent.com/B-UMMI/chewBBACA/master/CHEWBBACA/prodigal_training_files/Staphylococcus_aureus.trn \
+		-O $@ \
```

Comment:
Not sure if this `-O` is correct?

Reply:
Looks correct to me! Man-page says:
```
        -O file
       --output-document=file
           The documents will not be written to the appropriate files, but all will be concatenated together and written to file.
```

... and `$@` is the (output) target of the rule, so ...

Reply:
Yeah sorry I meant regarding the `$@`